src/main/java/com/icthh/xm/uaa/service/AccountService.java

```diff
@@ -240,4 +268,12 @@ private UserDTO buildUserDtoWithLogin(AuthorizeUserVm authorizeUserVm) {
         userDTO.addUserLogin(authorizeUserVm.getLogin());
         return userDTO;
     }
+
+    private String getNotificationKeyByChannel(OtpChannelType chanelType) {
+        switch (chanelType) {
+            case SMS: return TWILIO_MESSAGE_TYPE.toLowerCase();
```

Comment:
It have to be configurable
vf using smpp chanelType to send sms

Reply:
fixed